okio-fakefilesystem/src/commonMain/kotlin/okio/fakefilesystem/FakeFileSystem.kt

```diff
@@ -218,8 +238,13 @@ class FakeFileSystem(
     if (element !is File) {
       throw IOException("not a file: $file")
     }
+    if (!allowReadsWhileWriting) {
+      openFileOrNull(canonicalPath, operation = WRITE)?.let {
```

Comment:
nit: open as a adjective (is open) instead of a verb (to open) threw me initially. I thought this was opening the file.

Reply:
Renamed to findOpenFile.